core/constants.py

```diff
@@ -3,14 +3,21 @@
 import math
 import json
 
+# Autonomy General Configuration
+EVENT_LOOP_DELAY = 0.1
+
 # Navigation Parameters
 WIDTH = 640.0  # pixels
 FIELD_OF_VIEW = 40.0  # degrees
 TARGET_DISTANCE = 0.4  # meters
-RADIUS = .063  # meters
+RADIUS = 0.063  # meters
 SCALING_FACTOR = 10.0  # pixel-meters
-DRIVE_POWER = 200  # -1000 to 1000, normally 200 dropped lower for early testing to be safe
-WAYPOINT_DISTANCE_THRESHOLD = 1.5  # maximum threshold in meters between rover and waypoint
+DRIVE_POWER = (
+    250  # -1000 to 1000, normally 200 dropped lower for early testing to be safe
+)
+WAYPOINT_DISTANCE_THRESHOLD = (
+    1.5  # maximum threshold in meters between rover and waypoint
+)
```

Comment:
Why wrap these two, and only these two, constants in ( ... )

```suggestion
DRIVE_POWER = 250  # -1000 to 1000, normally 200 dropped lower for early testing to be safe
WAYPOINT_DISTANCE_THRESHOLD = 1.5  # maximum threshold in meters between rover and waypoint
```

Reply:
Yep